ui-guide/Components/TextFields.purs

```diff
@@ -87,8 +87,11 @@ cnDocumentationBlocks =
             [ HH.text "Error" ]
           , FormField.field_
             { label: HH.text "Email*"
-            , helpText: Just "Add the email of the End Advertiser."
-            , error: Just "This field is required."
+            , helpText: [ HH.text "Add the email of the End Advertiser." ]
+            , error:
+              [ HH.ul_ $
+                HH.li_ <<< pure <<< HH.text <$> [ "This field is required", "Must be a valid email" ]
+              ]
```

Comment:
Here's the only place I added an example for how this change is useful, now we can do stuff like showing multiple errors in a list.